templates/components/page-header.html.twig

```diff
@@ -1,6 +1,12 @@
 <header class="w-full flex justify-between items-center">
     <h1 class="page-title">{{ title }}</h1>
-    {% if link_url|default(false) %}
```

Comment:
This change requires that you change the other parts that use:
components/page-header.html.twig
otherwise, they will not display the link